api/profile/profileRouter.js

```diff
@@ -381,4 +381,22 @@ router.get('/match/:id', authRequired, (req, res, next) => {
     });
 });
 
+router.post('/mentor-information/', (req, res, next) => {
```

Comment:
Once we get Auth rewired let's please ensure that we lock down this endpoint with some `authRequired` middleware.

Before we merge this work let's capture that work w/ a TODO and also create a GH issue.

[TODO]: rewire w/ auth when the Auth team gets Auth-0 reinstalled.